lib/preprocessor.js

```diff
@@ -24,7 +24,13 @@ function executeProcessor (process, file, content) {
       }
     }
   })
-  return process(content, file, done) || donePromise
+
+  return (process(content, file, done) || Promise.resolve()).then((content) => {
+    if (content) {
+      return content
+    }
```

Comment:
`// process called done() (Either old sync api or an async function that did not return content)`